src/main/java/com/github/couchmove/utils/Utils.java

```diff
@@ -44,7 +44,7 @@ private static String initializeUserName() {
             try {
                 Class<?> c = Class.forName(className);
                 Method method = c.getDeclaredMethod(methodName);
-                Object o = c.newInstance();
+                Object o = c.getDeclaredConstructor().newInstance();
```

Comment:
Can you explain what this change is for ?

Reply:
It  was just removing deprecation warnings (like for the vars), it is deprecated since 9.
I can remove it since it's not related to the PR